contributors/design-proposals/local-storage-overview.md

```diff
@@ -475,7 +487,7 @@ Since local PVs are only accessible from specific nodes, the scheduler needs to
 Note: Block access will be considered as a separate feature because it can work for both remote and local storage.  The examples here are a suggestion on how such a feature can be applied to this local storage model, but is subject to change based on the final design for block access.
 
 1. The cluster that Bob uses has nodes that contain raw block devices that have not been formatted yet.
-2. The same addon DaemonSet can also discover block devices and creates corresponding PVs for them with the `block` field.  
+2. The same addon DaemonSet can also discover block devices and creates corresponding PVs for them. `path` is overloaded here to mean both fs path and block device path.
```

Comment:
The PersistentVolume object should have a volumeType field to indicate if it's a block volume or a filesystem volume.